internal/server/handlers/rbac/data_import_job.go

```diff
@@ -0,0 +1,30 @@
+package rbac
+
+import (
+	"context"
+	"errors"
+
+	everestv1alpha1 "github.com/percona/everest-operator/api/v1alpha1"
+	"github.com/percona/everest/pkg/rbac"
+)
+
+// ListDataImportJobs returns a list of DataImportJobs for the specified database clusters.
+func (h *rbacHandler) ListDataImportJobs(ctx context.Context, namespace, dbName string) (*everestv1alpha1.DataImportJobList, error) {
+	list, err := h.next.ListDataImportJobs(ctx, namespace, dbName)
+	if err != nil {
+		return nil, err
+	}
+	filtered := make([]everestv1alpha1.DataImportJob, 0, len(list.Items))
+	for _, job := range list.Items {
+		if err := h.enforce(ctx, rbac.ResourceDataImportJobs,
+			rbac.ActionRead, rbac.ObjectName(namespace, job.Spec.TargetClusterName),
+		); errors.Is(err, ErrInsufficientPermissions) {
+			continue
+		} else if err != nil {
+			return nil, err
+		}
+		filtered = append(filtered, job)
```

Comment:
are we going to limit access to particular DataImportJob via RBAC? If yes - what benefit does it bring to user?
I thought user shall have access to all of them for particular DBCluster or nothing.

Reply:
> are we going to limit access to particular DataImportJob via RBAC?

No, we are limiting access based on DBCluster name, same as DatabaseClusterBackups and DatabaseClusterRestores

> If yes - what benefit does it bring to user?

None actually, that's why we're not doing that

> I thought user shall have access to all of them for particular DBCluster or nothing.

Isn't this what we're doing? Please see the above line 20 and the unit test cases for it

Reply:
in such a case you don't need to check RBAC for each DataimportJob - just check `read` permission to DBCluster - if OK -> pass the whole list to result, if no - pass empty list

Reply:
Ah, okay, now I see what you mean - yes, it makes sense, the result is already a list of jobs for that databasecluster, so we don't have to check that in the loop. Just once is enough, so we return either the whole list or nothing. Thanks 👍🏻